llvm/include/llvm/CodeGen/MachineModuleInfo.h

```diff
@@ -106,16 +106,18 @@ class MachineModuleInfo {
   const Function *LastRequest = nullptr; ///< Used for shortcut/cache.
   MachineFunction *LastResult = nullptr; ///< Used for shortcut/cache.
 
-  MachineModuleInfo &operator=(MachineModuleInfo &&MMII) = delete;
+  /// Deleted copy constructor
```

Comment:
comment is not useful, I'd either say why it's deleted or just remove the comment